app/src/main/java/pro/vteam/freemarket/adapter/CategoriesPagerAdapter.java

```diff
@@ -0,0 +1,37 @@
+package pro.vteam.freemarket.adapter;
```

Comment:
rename `adapter` to `adapters` in the package name

Reply:
or even better, create `pagerAdapters` and `recyclerAdapter` in child of `adapters` package